src/logic/graphSamples.ts

```diff
@@ -1,11 +1,32 @@
 /* eslint-disable */
-const samples = {
+const samples: { [key: string]: any } = {
+  medium: {
+    graph: [
+      [0, 2, 6, 0, 0, 0, 0],
+      [2, 0, 0, 5, 0, 0, 0],
+      [6, 0, 0, 8, 0, 0, 0],
+      [0, 5, 8, 0, 10, 15, 0],
+      [0, 0, 0, 10, 0, 6, 2],
+      [0, 0, 0, 15, 6, 0, 6],
+      [0, 0, 0, 0, 2, 6, 0],
+    ],
+    string: '0 2 6 0 0 0 0\n2 0 0 5 0 0 0\n6 0 0 8 0 0 0\n0 5 8 0 10 15 0\n0 0 0 10 0 6 2\n0 0 0 15 6 0 6\n0 0 0 0 2 6 0',
+  },
   simple: {
-    graph: [[0, 1, 1], [1, 0, 1], [1, 1, 0]],
-    string: '0 1 1\n1 0 1\n1 1 0',
+    graph: [
+      [0, 1, 1, 1], 
+      [1, 0, 1, 1], 
+      [1, 1, 0, 1], 
+      [1,1,1,0],
```

Comment:
If you're still using vim keybindings press == while in normal mode in this line.